compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/AMDAIEDmaUtilsTest.cpp

```diff
@@ -271,6 +271,97 @@ TEST_F(AccessPatternCombinationTest, FailCombineAccessPatterns) {
                              {128, 1}, {32, 0}, {96, 64}, {128, 1}, 4, false);
 }
 
+class FoldLinearDimsTest : public ::testing::Test {
+ protected:
+  FoldLinearDimsTest() : rewriter(&context), loc(UnknownLoc::get(&context)) {
+    context.loadDialect<arith::ArithDialect>();
+  }
+
+  SmallVector<OpFoldResult> toOpFoldResult(const SmallVector<int64_t> &values) {
```

Comment:
```suggestion
  SmallVector<OpFoldResult> toOpFoldResult(ArrayRef<int64_t> values) {
```

and elsewhere in this file (big nit)